### ISSUE TITLE
connection Arduino with Raspberry Pi 3

### ISSUE THREAD

**Issue by Stellaki** [radionics/OpenSourceRadionics#19]
The only real solution for the hotbits problem would be to generate hotbits in realtime ... which is only possible with the Raspberry Pi 3. 
Can we connect the Arduino with a Raspberry Pi 3 for the realtime use of  hotbits? 

Regards
Stella

**Comment by radionics** [comment]
Yes, but not the Arduino will connect to the Raspberry, but the AetherOne application, by using a http client.
